README.md

```diff
@@ -72,11 +72,92 @@ func main() {
 }
 ```
 
-For more information about FaunaDB query language, consult our query language
-[reference documentation](https://docs.fauna.com/fauna/current/reference/queryapi/).
+### Streaming feature usage
+```go
+package main
+
+import f "github.com/fauna/faunadb-go/v4/faunadb"
+
+func main() {
+	secret := ""
+	dbClient = f.NewFaunaClient(secret)
+	var ref f.RefV
+	value, err := dbClient.Query(f.Get(&ref))
+	if err != nil {
+		panic(err)
+	}
+	err = value.At(f.ObjKey("ref")).Get(&docRef)
+	var subscription f.StreamSubscription
+	subscription = dbClient.Stream(docRef)
+	err = subscription.Start()
+	if err != nil {
+		panic("Panic")
+	}
+	for a := range subscription.StreamEvents() {
+		switch a.Type() {
+	
+		case f.StartEventT:
+			// do smth on start event
+	
+		case f.HistoryRewriteEventT:
+			// do smth on historyRewrite event	
+			
+		case f.VersionEventT:
+			// do smth on version event
+			
+		case f.ErrorEventT:
+			// do smth on error event
+			subscription.Close() // if you want to close streaming on errors
+		}
+	}
+}
+```
+
+### Omitempty usage
+```go
+package main
+
+import f "github.com/fauna/faunadb-go/v4/faunadb"
+
+func main() {
+	secret := ""
+	dbClient = f.NewFaunaClient(secret)
+	var ref f.RefV
+	value, err := dbClient.Query(f.Get(&ref))
+	if err != nil {
+		panic(err)
+	}
+	type OmitStruct struct {
+		Name           string      `fauna:"name,omitempty"`
+		Age            int         `fauna:"age,omitempty"`
+		Payment        float64     `fauna:"payment,omitempty"`
+		AgePointer     *int        `fauna:"agePointer,omitempty"`
+		PaymentPointer *float64    `fauna:"paymentPointer,omitempty"`
+	}
+	_, err := dbClient.Query(
+		f.Create(f.Collection("categories"), f.Obj{"data": OmitStruct{Name: "John", Age: 0}}))
+	if err != nil {
+		panic(err)
+	}
+}
+```
+###Result (empty/zero fields will be ignored):
```

Comment:
```suggestion

**Result (empty/zero fields will be ignored):**

```

As a heading there needs to be a space between `#` and the text. But this result is part of the `omitempty` example, so a new section title is inappropriate.